objects/tokenList.py

```diff
@@ -268,28 +269,35 @@ def usersTimeoutCheckLoop() -> None:
         exceptions: list[Exception] = []
         timeoutLimit = int(time.time()) - OSU_MAX_PING_DELTA
 
-        for token in osuToken.get_tokens():
```

Comment:
need data to be fetched AFTER lock is held

Reply:
should fetch a list of token ids, loop thru, acquire lock and then fetch data